networking/src/Models/AnswerClasses.ts

```diff
@@ -36,7 +37,7 @@ export abstract class BaseAnswer<T> {
       this.answerType = answerType;
   }
 
-  abstract isNormAnswerUnique(otherNormAnswers: T[]): Boolean;
+  abstract isEqualTo(otherNormAnswers: T[]): Boolean;
```

Comment:
Flips naming back to `isEqualTo`. Currently, this function is only called when a `ShortAnswerResponse` is submitted and the equality is run on an answer by answer basis, so I think it makes sense to compare one answer to another answer via `isEqualTo`.